src/hyphenClient.ts

```diff
@@ -0,0 +1,74 @@
+import type { EvaluationResponse, HyphenEvaluationContext } from './types';
+import NodeCache from '@cacheable/node-cache';
+import { horizon, cache } from './config';
+
+export class HyphenClient {
+  private readonly publicKey: string;
+  private readonly horizonServerUrls: string[];
+  private cache: NodeCache;
+
+  constructor(publicKey: string, horizonServerUrls?: string[]) {
```

Comment:
do the default here for horizonUrls... it will show up better in intellisense that way.